### DISCUSSION TITLE
Only good for User Data Backups or also for Linux Root Filesystem Snapshots?

### DISCUSSION THREAD

**Discussion by Tom-H-L** [borgbackup/borg#7115] [Q&A] [answered]
Can someone use Borg only for user data backups or also for system file snapshots of the Linux root files system, or is it better to use another tool for that task, e.g. "Back in Time"?

**Comment by infectormp** [comment, marked as answer]
You can read FAQ article about this https://borgbackup.readthedocs.io/en/stable/faq.html#can-i-backup-my-root-partition-with-borg
